onnx/defs/quantization/defs.cc

```diff
@@ -101,4 +102,49 @@ ONNX_OPERATOR_SET_SCHEMA(
               updateOutputShape(ctx, 0, input_shape);
             }));
 
+static const char* FusedQuantizeLinear_ver11_doc = R"DOC(
+A Function to fuse calculation for Scale, Zero Point and FP32->8Bit convertion of FP32 Input data.
+Outputs Scale, ZeroPoint and Quantized Input for a given FP32 Input)DOC";
+
+ONNX_OPERATOR_SET_SCHEMA(
+    FusedQuantizeLinear,
+    11,
+    OpSchema()
+        .SetDoc(FusedQuantizeLinear_ver11_doc)
+        .Input(0, "X", "Input tensor", "T1")
+        .Output(0, "Y", "Quantized output tensor", "T2")
+        .Output(1, "Y_Scale", "Output Scale. It's a scalar or a 1D tensor with size 1.", "tensor(float)")
+        .Output(2, "Y_Zero_Point", "Output Zero point. It's a scalar or a 1D tensor of size 1.", "T2")
+        .Attr(
+          "to",
+          "The data type to which the elements of the input tensor are quantized to. Strictly must be one of the types from DataType enum in TensorProto",
+          AttributeProto::INT,
+          static_cast<int64_t>(2))
+        .TypeConstraint(
+          "T1",
+          {"tensor(float)"},
+          "Constrain 'X' to float tensor.")
+      .TypeConstraint(
+          "T2",
+          {"tensor(uint8)"},
+          "Constrain 'Y_Zero_Point' and 'Y' to 8-bit unsigned integer tensor.")
+      .FunctionBody(FunctionBodyHelper::BuildNodes(
+          {// nodes: {outputs, op, inputs, attributes}
+           FunctionBodyHelper::Const<float>("Q_Min", 0.0f),
+           FunctionBodyHelper::Const<float>("Q_Max", 255.f),
+           {{"X_Min"}, "ReduceMin", {"X"}},
+           {{"X_Min_Adjusted"}, "Min", {"X_Min", "Q_Min"}},
+           {{"X_Max"}, "ReduceMax", {"X"}},
+           {{"X_Max_Adjusted"}, "Max", {"X_Max", "Q_Min"}},
```

Comment:
Is that supposed to be Q_Min or Q_Max?

Reply:
It is supposed to be Q_Min... What we are doing here is making sure 0 is included in the input range to guarantee that zero can be uniquely represented in the quantized range.
Example: if input range is [-3, -1] then this would adjust max to 0 and the new range will be [-3, 0]